src/Our.Umbraco.EnvironmentDashboard/Components/EnvironmentDashboardGroupsCollectionBuilder.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Umbraco.Core.Composing;
+
+namespace Our.Umbraco.EnvironmentDashboard.Components
+{
+	public class EnvironmentDashboardGroupsCollectionBuilder : OrderedCollectionBuilderBase<EnvironmentDashboardGroupsCollectionBuilder, EnvironmentDashboardGroupsCollection, IDashboardGroupsProvider>
+	{
+		protected override EnvironmentDashboardGroupsCollectionBuilder This => this;
+	}
```

Comment:
I'm not entire sure what this is for, as I haven't used it. So I'm not sure why it needs to be Group**s**, and not just singlular Group

The words Environment and Dashboard appear a lot, which can cause some confusion, so I would probably rename EnvironmentDashboardGroupsCollectionBuilder to something like InfoGroupCollectionBuilder

Reply:
This is the builder for the IDashboardGroupsProviders

Reply:
Perhaps call this something shorter...
- InfoGroupCollectionBuilder
- DetailGroupCollectionBuilder
- GroupCollectionBuilder
- PanelCollectionBuilder
- CategoryCollectionBuilder

I'm kind of leaning toward InfoGroupCollectionBuilder or DetailGroupCollectionBuilder

Thoughts?